spec/customer_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'customer'
+
+describe Customer do
+  let(:subject) {described_class.new("Piers", "+447736409366")}
+  let(:restaurant) {double :restaurant}
+  let(:orderClass) {double (:orderClass), :new => order}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing inside {.<br>Space missing inside }.